src/AutoRest.CSharp/LowLevel/Generation/LowLevelClientWriter.cs

```diff
@@ -111,6 +125,22 @@ public void WriteClient()
             }
         }
 
+        private void WriteFuncBodyWithProcessMessage(CodeWriter writer, CodeWriterDeclaration messageVariable, RestClientMethod operation, string pipelineName, bool async)
```

Comment:
It seems that this the function is the same as the one in RestClientWriter. Can we use one?

Reply:
Which one you mean?

Reply:
OK. You removed `WriteFuncBodyWithProcessMessage` function from RestClientWriter in latest commit. The issue was resolved.